graphqlapi/src/main/java/io/stargate/graphql/web/resources/schemafirst/FilesResource.java

```diff
@@ -38,6 +50,27 @@ public Response getCqlDirectives() {
         .build();
   }
 
+  @GET
+  @Path("/namespace/{namespaceName}.graphql")
```

Comment:
I know I'm late to the party but do we think "namespace" used in graphql will be confused with "namespace" in the docsapi?

Reply:
I picked it because it's essentially the same thing: a thin abstraction over a CQL keyspace. But I don't mind changing it if you think we should use something different.

Btw you're not late, there's still time to improve naming and in fact I planned to do so before we merge the main PR (starting with the URIs).

Reply:
Cool, I think that makes sense. We probably just need to be cognizant of the double meaning when documenting or discussing. `namespace` is probably a more familiar term for those using graphql as well.